Directory.Packages.props

```diff
@@ -188,7 +163,69 @@
     <PackageVersion Include="Azure.Identity" Version="1.13.1" />
     <!-- https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3313 -->
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageVersion Include="System.Formats.Asn1" Version="$(SystemFormatsAsn1PackageVersion)" />
-    <PackageVersion Include="System.Text.Json" Version="$(SystemTextJsonPackageVersion)" />
+  </ItemGroup>
+
+  <!-- The following 2 groups are for packages that need to switch based on the .NET TFM being used. -->
+
+  <ItemGroup>
```

Comment:
NIT: Should this be conditioned as well for net8 tfm?

Reply:
My thinking was that these are unconditional, and then the "STS" version "updates" the PackageVersion. Either way could work, I don't have strong opinion. Do you?